_developer-files/connect/api/objects/form-properties/sources/saas/outreach-object.md

```diff
@@ -0,0 +1,50 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+product-type: "connect"
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-outreach-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "Outreach Source Form Property"
+api-type: "platform.outreach"
+display-name: "Outreach"
+
+source-type: "saas"
+docs-name: "outreach" # This should be whatever integration.name is. Ex: LinkedIn Ads is linkedin-ads
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-start-date: true
+
+# Only source-specific attributes need to be listed here.
+# The following attributes are considered common,
+# and therefore don't need to be listed:
+# anchor_time, cron_expression, frequency_in_minutes, image_version, start_date 
+
+object-attributes:
+  - name: "quota_limit"
+    type: "string"
+    required: false
+    description: |
+      The percentage of your standard API quota Stitch is allowed to use. Refer to the [{{ form-property.display-name }} documentation]({{ doc-link | append: "#add-integration" }}) for more information on defining this limit.
+    value: "XX%"
```

Comment:
I'd be surprised if a user had to include the percent sign when submitting this to the API.

To test this yourself, what you can do is:

1. Create and set up the source in the Stitch app
2. Fill out the field(s) in question
3. Use the API to retrieve the source's data (This would be the **GET /v4/sources/{source_id}** endpoint, not **GET /v4/source-types/{source_type}** like we usually use)

This will allow you to see what the UI expects the value to be and how it saves it to the database, which is where the API will retrieve it from. For example: If, in the Stitch app, you input `10` into the Quota Limit field, you should see `quota_limit: "10"` when you retrieve the source.

Let me know if you want to walk through this IRL.